_visualisation/core/focus-depth.md

```diff
@@ -0,0 +1,80 @@
+* Focus on selected Subjects (at the starts of Chains), and trace forwards all the Objects they are interested in.
+
+#### Object Focus
+
+![BOVS Diagram with Object Focus](/visualisation/diagrams/bovs-core-focus-object.png){: style="width: 87.5%" }
+
+Diagrams with an Object Focus (the most common) show all the Parties that exist in the reverse direction of the Arrows from the selected Objects (usually just one). For instance, it may trace all of the Parties interested in a specific Company, back to a set of Persons.
+
+#### Subject Focus
+
+![BOVS Diagram with Subject Focus](/visualisation/diagrams/bovs-core-focus-subject.png){: style="width: 87.5%" }
+
+Diagrams with a Subject Focus show all the Parties that exist in the forward direction of the Arrows from the selected Subjects (usually just one). For instance, it may show all the interests of a specific Person.
+
+### Interests
+
+**Normally, BOVS diagrams show both ownership and control interests,** that is, they lack an Interest Focus.
+
+#### Interest Focus
+
+![BOVS Diagram with Interest Focus](/visualisation/diagrams/bovs-core-focus-interest.png){: style="width: 87.5%" }
+
+You may choose to adopt an Interest Focus, that is, include only one type of interest. If so:
+
+* Only draw Arrows and Nodes for Parties with your chosen type of interest (known or [unknown](/visualisation/core/unknowns)).
+* Clearly indicate on your diagram that it shows only your chosen type of interest.
+
+
+## Depth
+
+**Diagrams have a Depth, which, unless you decide and note otherwise, is unlimited.** All Chains in the diagram are cut short at a length equal to the Depth.
+
+In **Object Focus** diagrams, the Chains in the reverse direction from the Entities at the end should be cut short after the specified Depth. This allows you to focus on:
+
+* The ultimate Beneficiaries of the Entity you are focusing on (unlimited Depth)
+* The Parties interested in the Entity through a given number of intermediaries (Depth X)
+* The immediate Beneficiaries of the Entity only (Depth 1)
+
+In **Subject Focus** diagrams, the Chains in the forward direction from the Subjects at the start should be cut short after the specified Depth. This allows you to show either:
+
+* All the Entities that the Subjects you are focusing on have an interest in, through any number of intermediaries (unlimited Depth)
+* All the Entities that the Subjects have an interest in, through a given number of intermediaries (Depth X)
+* The Direct interests of the Subjects only (Depth 1)
+
+Depth should be limited only for the above reasons, that is, not for cosmetic reasons, to make diagrams fit into a given space, or to hide Parties not relevant to your point. If this is your chief concern, use [Stacks](/visualisation/core/stacks) rather than limiting your diagram's Depth.
+
+### Appearance
+
+![BOVS Diagram with Depth limited to 1](/visualisation/diagrams/bovs-core-focus-depth.png){: style="width: 87.5%" }
```

Comment:
The diagram could potentially communicate the rule more clearly if instead of 'Entity D' it had 'Person A' with no onward dotted line.

This would show that:

* A diagram may include chains of different lengths
* Only the lengths > depth need a ... line